src/witan/workspace_api/schema.clj

```diff
@@ -1,5 +1,6 @@
 (ns witan.workspace-api.schema
-  (:require [schema.core :as s]))
+  (:require [clojure.set]
```

Comment:
We're seen this issue in the past and it occurred today - things depending on this library fail to compile it due to `java.lang.ClassNotFoundException: clojure.set` ([example](https://gist.github.com/Eleonore9/e6aa8b2266564c134834382581176eb4))